changelog/7760.breaking.rst

```diff
@@ -0,0 +1,2 @@
+Replaced the use of scikit-image with ContourPy within `sunpy.map.Map.contour`.
+There is no change to returned contours but offers a faster way to contour.
```

Comment:
```suggestion
`sunpy.map.GenericMap.contour` now internally uses ContourPy instead of scikit-image for improved performance.
The API for the method has not changed, but as before, additional keyword arguments are passed through to the underlying function, so some such keyword arguments may no longer be supported.
```

Reply:
This also fixes the broken ref